source/includes/steps-view-logs.yaml

```diff
@@ -0,0 +1,83 @@
+---
+title: "Find your Session ID."
+level: 4
+ref: session-id
+content: |
+
+  .. example::
+
+     |mdb-shell| displays the session ID each time you open the shell.
+
+     .. code-block:: javascript
+        :copyable: false
+        :emphasize-lines: 3
+
+        $ mongosh
+
+        Current sessionID: c2961dbd6b73b052671d9df0
+        Connecting to: mongodb://127.0.0.1:27017
+        Using MongoDB: 4.2.8
```

Comment:
We've changed the greeting message a bit with `0.0.6`. I am not sure how accurate you'd want to be, but this is what it looks like now:
```shell
Current sessionID:  c6d052d327b75c50c285d30f
Connecting to:      mongodb://127.0.0.1:27017
Using MongoDB:      4.2.5
Using Mongosh Beta: 0.0.6
```

Reply:
updated